Source/Math/NcclComm.cpp

```diff
@@ -0,0 +1,95 @@
+//
+// Copyright (c) 2016, NVIDIA CORPORATION. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
```

Comment:
Please include the guards in NcclComm.cpp, we follow that pattern elsewhere, too.